airbyte-webapp/.eslintrc

```diff
@@ -16,6 +17,15 @@
   },
   "rules": {
     "curly": "error",
+    "css-modules/no-undef-class": [2, { "camelCase": true }],
+    "css-modules/no-unused-class": [2, { "camelCase": true }],
+    "no-restricted-imports": [1, {
```

Comment:
Prefer to use "error" and "warn" here instead of 2 and 1. I would recommend to use "warning" if it's fixable and save "error" for very serious issues

Reply:
Sorry, I was mistaken in my comment. the string is `warn` not `warning`